src/backend/core/models.py

```
@@ -972,6 +972,80 @@ def clean(self):
+        help_text=_("Service this account is associated with"),
+    )
+    
+    roles = models.JSONField(
+        _("roles"),
+        default=list,
+        blank=True,
+        help_text=_("Array of role strings for this account"),
+    )
+
+
+class Account(BaseModel):
```

Ruff (0.14.10)

Model does not define `__str__` method

(DJ008)